### PR TITLE
Cleanup code formatting

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: presto-cli/src/test/java/com/facebook/presto/cli/TestConsoleHistory.java [hunk 1]
```diff
@@ -25,7 +25,8 @@
 public class TestConsoleHistory
 {
     @Test
-    public void testNonExistingHomeFolder() throws Exception
+    public void testNonExistingHomeFolder()
+            throws Exception
```
[the remark this answers is not in the document]
Comment (electrum) [reply presto-cli/src/test/java/com/facebook/presto/cli/TestConsoleHistory.java]: I've been unable to find a Checkstyle rule for that.

[the remark this answers is not in the document]
Comment (kokosing) [reply presto-cli/src/test/java/com/facebook/presto/cli/TestConsoleHistory.java]: It should be easy to write a regex for that.

File: presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/BingTile.java [hunk 2]
```diff
@@ -51,9 +51,9 @@ public boolean equals(Object other)
         }
 
         BingTile otherTile = (BingTile) other;
-        return this.x == otherTile.x
-            && this.y == otherTile.y
-            && this.zoomLevel == otherTile.zoomLevel;
```
[the remark this answers is not in the document]
Comment (electrum) [reply presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/BingTile.java]: Filed https://github.com/airlift/airbase/issues/158

File: presto-cli/src/test/java/com/facebook/presto/cli/TestConsoleHistory.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (electrum) [reply presto-cli/src/test/java/com/facebook/presto/cli/TestConsoleHistory.java]: Filed https://github.com/airlift/airbase/issues/160

[the remark this answers is not in the document]
Comment (kokosing) [reply presto-cli/src/test/java/com/facebook/presto/cli/TestConsoleHistory.java]: Thanks

File: presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/BingTile.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (kokosing) [reply presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/BingTile.java]: thanks

**Pull request merged**